gravityflow/gravityflowwoocommerce
# Fix an issue where entries cannot be released from a Checkout step

re: [HS#11573](https://secure.helpscout.net/conversation/1008290715/11573?folderId=2308455)

This PR fixes an issue where entries cannot be released from a Checkout step. This happened in two scenarios:

1. When users enable the WooCommerce feature that allows new accounts being created during the checkout process. 
2. When adding the `add-to-cart` param to the  `{workflow_woocommerce_checkout_url}` merge tag (this can only be reproduced in some edge cases, for example, the customer's server).

## Testing instructions
1. Create a very simple form with just an email field.
2. Adding a WooCommerce Checkout step to the workflow and set the "Email" field as the assignee.
3. In the WooCommerce Checkout message, change the merge tag to `{workflow_woocommerce_checkout_url}&add-to-cart=123` (please use a real product id).
4. Set up your WooCommerce Settings - Accounts & Privacy as: https://www.dropbox.com/s/cps4n0turdqxrsq/Screenshot%202020-01-01%2014.08.55.png?dl=0
5. Submit the form and following the email to checkout. With this branch, the entry should be released correctly.
